bot/exts/recruitment/talentpool/_review.py

```diff
@@ -104,10 +118,7 @@ async def post_review(self, user_id: int, update_database: bool) -> None:
         )
 
         review = "\n\n".join(part for part in (opening, current_nominations, review_body, vote_request))
```

Comment:
```suggestion
        review = "\n\n".join((opening, current_nominations, review_body, vote_request))
```

Reply:
That's isn't my code :pensive: but thanks haha